### ISSUE TITLE
Devoid card aesthetics

### ISSUE THREAD

**Issue by ShawnieBoy** [magarena/magarena#580]
Currently devoid card frames are left white due to the original frames being transparent for card images.

Token images which have transparent images (Eldrazi Spawn, Eldrazi Scion) use usual colorless frame for colorless cards.

Should Devoid use the colorless frame as well, or should they be white? If so should all transparency be white, and so Eldrazi Spawn and Scion be white, along with overlay cards?

Images to illustrate:

Red Devoid Colorless
![red-devoid](https://cloud.githubusercontent.com/assets/2363180/11932719/5d437fc0-a7ef-11e5-9e2a-70b5b8cb8273.jpg)

Red Devoid White/Clear
![red-white](https://cloud.githubusercontent.com/assets/2363180/11932724/61587f7a-a7ef-11e5-91cb-340c887cf521.jpg)

Colorless Token
![colorless token](https://cloud.githubusercontent.com/assets/2363180/11932728/6569439c-a7ef-11e5-98c0-e2885edbca17.jpg)

White/Clear Token
![clear token](https://cloud.githubusercontent.com/assets/2363180/11932785/b99c288a-a7ef-11e5-813b-6af98282b9fb.jpg)

**Comment by ShawnieBoy** [comment]
It would appear that previous colorless tokens which used a colorless frame now use a clear frame - Does push this more towards 'clear' frames, as long as they wouldn't get confused with White frames.

**Comment by ShawnieBoy** [comment]
Being able to use overlays (I have overlays for devoid, but not yet for Nyx or Miracle) Would make colorless cards as a base more likely.

@melvinzhang @lodici - Any thoughts on which way to go with transparent frames?

**Comment by melvinzhang** [comment]
@ShawnieBoy could you summarize what are the options we are considering?

**Comment by lodici** [comment]
> Any thoughts on which way to go with transparent frames?

Sorry, a lot of what you guys talk about regarding frames and stuff goes right over my head. What ways are you proposing again?

**Comment by ShawnieBoy** [comment]
When dealing with transparency in relation to colorless cards:

Should the transparency be represented as an absence of color: ie - white

![](https://cloud.githubusercontent.com/assets/2363180/11932724/61587f7a-a7ef-11e5-91cb-340c887cf521.jpg)

Or should the transparency be represented as a colorless frame

![](https://cloud.githubusercontent.com/assets/2363180/11932719/5d437fc0-a7ef-11e5-9e2a-70b5b8cb8273.jpg)

**Comment by lodici** [comment]
Oh, Devoid card ae __s__ thetics! I thought aethetics was some sort of MTG frame variation. Still not really the wiser. Sorry to be blunt but why does it matter which method you use?

**Comment by melvinzhang** [comment]
For the first case, wouldn't a white devoid be all white? I think the colorless frame (option 2) is less confusing.

**Comment by ShawnieBoy** [comment]
hehe - thanks @lodici 

I guess in a way it doesn't matter overly much, but I'm having trouble reaching a decision. It's mainly due to eldrazi, morph, colorless sliver etc. using transparent frames to represent colorless - as full card-size art for these images will be impossible to obtain to accurately represent the card images - a workaround similar to the way planeswalker frames are transparent is needed.

If having all colorless transparency treated the same gives a consistency.

Thanks for the feedback - @melvinzhang, white cards and white-for-transparency can cause problems.

**Comment by ShawnieBoy** [comment]
![wcard](https://cloud.githubusercontent.com/assets/2363180/12091046/eba5c8ca-b2eb-11e5-8d47-c094fe905014.jpg)
![ccard](https://cloud.githubusercontent.com/assets/2363180/12091080/24de5f58-b2ec-11e5-9709-1dc1d2a8dd68.jpg)

I'll go forwards with all transparency for colorless will become colorless frames
